app/components/Login.jsx

```diff
@@ -4,34 +4,36 @@ import React from 'react';
 
 import TextField from 'material-ui/TextField';
 import Button from 'material-ui/RaisedButton';
-
+import Paper from 'material-ui/Paper';
 
 export const Login = ({ login }) => (
```

Comment:
Login doesn't have a header while signup does.  Signup is also a lot taller than login and is the lines look double spaced on signup but not in login.  Try to just pick one format between the two of them?